lib/Yars/Routes.pm

```diff
@@ -82,6 +82,13 @@ sub _get {
           || _redirect_to_remote_stash( $c, $filename, $md5 )
           || $c->reply->not_found;
     };
+
+    if ($c->req->headers->header('X-Use-X-Accel'))
+    {
+        $c->res->headers->add('X-Accel-Redirect', "$dir/$filename");
```

Comment:
I think it may be okay, if ordered in the configuration correctly (?) because the real disk files probably won't overlap with the /file/md5/filename or /file/filename/md5, but I would prefer to have an explicit /static in the URI, in case we need to add extra routes that might in the future conflict.